internal/policies/mount/mount.go

```diff
@@ -374,10 +374,15 @@ func parseEntryValues(ctx context.Context, e entry.Entry) (p []string, err error
 			continue
 		}
 
+		if _, ok := seen[v]; ok {
+			log.Debugf(ctx, i18n.G("Value %q is duplicated."), v)
+			continue
+		}
+
 		// Compares "normal" and prefixed values the same way, since the unit name will be the same.
 		tmp := strings.TrimPrefix(v, krbTag)
 		if _, ok := seen[tmp]; ok {
-			log.Debugf(ctx, i18n.G("Value %q is duplicated. Tagged values are the same as untagged ones."), v)
+			log.Warningf(ctx, i18n.G("The location %q was already set up to be mounted. Authenticated and anonymous mounts are considered the same."), tmp)
```

Comment:
I would say "Authenticated and anonymous mounts are considered the same." -> "Only the first mount setting will be considered".

or something in more proper english which was say exactly which option (that is in %q, I agree) is considered.

Reply:
I took 30 minutes to figure out a way to write this

Reply:
The issue here is the `tmp` might be the entry with krb5, and it was stripped before getting into seen, so telling %q was tmp is misleading…

Reply:
OR, you store `seen[tmp] = v`. that way, we have the initial value…

Reply:
That's why I'm trying to add this message that tagged and untagged are the same under our eyes... If I print the tagged value, it could create confusion just the same...

Reply:
Yes, but in the future, it might be other things, and "tag" and just an internal naming, no?

I would be really bare then, as we can get the initial value, and just telling "considering this value and ignoring that one", that way, it’s clear to the sysadmins as they put both values somewhere.

(but for that, we need seen to be a map[string][string]